From 257965dffda952c54ae431feeb64326ac3f9d055 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:17:54 -0400
Subject: [alsa] Fix typos in alsa-driver-api.xml

Message-id: <140972823573-94-patchset-jkysela@redhat.com>
Patchwork-id: 90794
O-Subject: [RHEL 7.1 PATCH 093/442] ALSA: Fix typos in alsa-driver-api.xml
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

This patch fixed 2 typos in DocBook/alsa-driver-api.xml.
It is because this file is generated by make xmldocs,
I have to fix typos within source files.

Signed-off-by: Masanari Iida <standby24x7@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 0a11458cbbb81d044c8fc5a77a7584cd33f08c48)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/core/pcm_native.c b/sound/core/pcm_native.c
index 01a5e05..10b5d1a 100644
--- a/sound/core/pcm_native.c
+++ b/sound/core/pcm_native.c
@@ -954,7 +954,7 @@ static struct action_ops snd_pcm_action_stop = {
  *
  * The state of each stream is then changed to the given state unconditionally.
  *
- * Return: Zero if succesful, or a negative error code.
+ * Return: Zero if successful, or a negative error code.
  */
 int snd_pcm_stop(struct snd_pcm_substream *substream, snd_pcm_state_t state)
 {
diff --git a/sound/core/rawmidi.c b/sound/core/rawmidi.c
index 7b596b5..f016be7 100644
--- a/sound/core/rawmidi.c
+++ b/sound/core/rawmidi.c
@@ -1101,7 +1101,7 @@ int snd_rawmidi_transmit_peek(struct snd_rawmidi_substream *substream,
 /**
  * snd_rawmidi_transmit_ack - acknowledge the transmission
  * @substream: the rawmidi substream
- * @count: the tranferred count
+ * @count: the transferred count
  *
  * Advances the hardware pointer for the internal output buffer with
  * the given size and updates the condition.
-- 
1.7.1